packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/cli.integtest.ts

```diff
@@ -50,6 +50,31 @@ integTest('VPC Lookup', withDefaultFixture(async (fixture) => {
   await fixture.cdkDeploy('import-vpc', { modEnv: { ENABLE_VPC_TESTING: 'IMPORT' } });
 }));
 
+integTest(
+  'cdk migrate typescript',
+  withTemporaryDirectory(
+    withPackages(async (context) => {
+
+      const tempShell = ShellHelper.fromContext(context);
+      await context.packages.makeCliAvailable();
+
+      const tempPath = path.resolve(context.integTestDir);
+      const inputFile = path.join(__dirname, 'template.txt');
+
+      await tempShell.shell([
```

Comment:
This can't possibly pass. It's missing `--stack-name`.

Reply:
Hm good point. It's actually kind of alarming because this did pass, and that likely means our other init tests aren't being tested currently.

Reply:
Most of those tests have an `expect` statement of some sort. I see that not all do but I'm not totally familiar with their inner workings.